### ISSUE TITLE
Wrong html entities in <title> tag after PR 7463

### ISSUE THREAD

**Issue by jagbarcelo** [OrchardCMS/OrchardCore#7497]
We have spotted an issue with [PR 7463](https://github.com/OrchardCMS/OrchardCore/pull/7463).

Before this change, our <title> tags with special characters were rendered just fine (plain raw text). Now we have them shown as hex html entities but the **&** character is being escaped, so they are not rendered properly:

![image](https://user-images.githubusercontent.com/6704109/97969705-b99e0780-1dc0-11eb-986f-cba860b518e6.png)

If you go to the actual html output of the page it has:

![image](https://user-images.githubusercontent.com/6704109/97970402-c53dfe00-1dc1-11eb-8c82-b47bb0014d91.png)

but it should have:
`<title>P&#xE1;gina de prueba - my-site</title>`

Summing up, the **&** character in the `&#xE1;` entity should not be escaped to `&amp;xE1;`

BTW, the <title> tag in TheBlogTheme is rendered with (which is fine):
`<title>{{ "PageTitle" | shape_new | shape_stringify }}</title>`

//cc @jtkech

**Comment by hishamco** [comment]
Encoding issue, I will fix this

**Comment by sebastienros** [comment]
How is your title rendered since you mention that it's fine in the Blog Theme with `<title>{{ "PageTitle" | shape_new | shape_stringify }}</title>`

**Comment by jtkech** [comment]
See my last comment in #7498

**Comment by jagbarcelo** [comment]
I meant that TheBlogTheme is correct in using `<title>{{ "PageTitle" | shape_new | shape_stringify }}</title>`, not that the output of that was correct (which was not).

**Comment by sebastienros** [comment]
@jagbarcelo thanks for the confirmation. Checking @jtkech 's answer now.
